<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CyberslackingClass</class>
 <widget class="QWidget" name="CyberslackingClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cyberslacking</string>
  </property>
  <widget class="QGroupBox" name="worktime_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>今日时长</string>
   </property>
   <widget class="QLineEdit" name="work_time_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_duration_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="slack_time_lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total_duration_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总时长</string>
    </property>
   </widget>
   <widget class="QLabel" name="work_time_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>搬砖时长</string>
    </property>
   </widget>
   <widget class="QLabel" name="slack_time_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>摸鱼时长</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label_1">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="equal_label_1">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="income_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>今日收益</string>
   </property>
   <widget class="QLineEdit" name="work_income_lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="total_income_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="slack_income_lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="total_income_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>总收益</string>
    </property>
   </widget>
   <widget class="QLabel" name="work_income_label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>搬砖收益</string>
    </property>
   </widget>
   <widget class="QLabel" name="slack_income_label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>摸鱼收益</string>
    </property>
   </widget>
   <widget class="QLabel" name="add_label_2">
    <property name="geometry">
     <rect>
      <x>176</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="equal_label_2">
    <property name="geometry">
     <rect>
      <x>87</x>
      <y>40</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="unit_work_income_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>单位搬砖收益</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="unit_work_income_lineEdit">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unit_label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>元/小时</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="offwork_pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>265</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>下班</string>
   </property>
  </widget>
  <widget class="QPushButton" name="init_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>235</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>初始化信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="work_pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>235</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>上班</string>
   </property>
  </widget>
  <widget class="QPushButton" name="current_pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>265</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>当前信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="work_radioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>搬砖</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="slack_radioButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>摸鱼(休息)</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cyberslacking.qrc"/>
 </resources>
 <connections/>
</ui>
